test/models/onnx/onnx_prototxt_converter.py

```diff
@@ -0,0 +1,87 @@
+# *****************************************************************************
+#  Copyright 2017-2019 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# *****************************************************************************
+"""Converts protobuf files from binary format into text format and vice-versa.
+
+Supports files with only '.onnx' or '.prototxt' extensions.
+
+Usage:
+  onnx_prototxt_converter.py INPUT_FILE [OUTPUT_FILE]
```

Comment:
Please add a sentence which describes what happens if `OUTPUT_FILE` is not specified.